qunit-dom/package.json

```diff
@@ -5,7 +5,6 @@
   "keywords": [
     "browser",
     "dom",
-    "ember-addon",
```

Comment:
> Why'd we want to remove this keyword?

we can't have `ember-addon` and have a vanilla ESM package.
To use `ember-addon` currently requires:
- we use addon-main.cjs
- include `@embroider/addon-shim`

which is a lot more work for the build system than we need.

So, qunit-dom is now a vanilla npm package, no emberisms

Reply:
nay, things start failing with the removal of `ember-addon`, specifically in embroider builds.

Reply:
and ember-cli keys off this keyword as well.
which... _isn't good_.

Reply:
when using `ember-addon`, we are required to have a v1-addon entrypoint. this is typically the `index.js` in v1 addons, or the `addon-main.cjs` for v2 addons. we don't need nor want either

Reply:
oh, and we'd have to remove `type=module`, and remove publint / are-the-types-wrong as well, since embroider doesn't support type=module for ember-v2-addons yet.

Reply:
That's a bit unfortunate, makes sense though :sweat_smile: